Casks/latest.rb

```diff
@@ -1,8 +1,8 @@
 cask "latest" do
-  version "0.7.3,462"
+  version "0.8,487"
   sha256 :no_check
 
-  url "https://max.codes/latest/download"
+  url "https://max.codes/latest/Latest.zip"
```

Comment:
```suggestion
  url "https://max.codes/latest/#{version}.zip"
```
The sparkle feed revealed https://max.codes/latest/0.8.zip, so we can switch to the versioned `url`. This requires setting `sha256` as well.

Reply:
I am sorry for my late response. `version` currently includes the build number as well, which is not part of the URL. 0.8.zip and Latest.zip point to the same zip file, so I _think_ it would be fine to keep the generic url? I can add the sha256 string nonetheless.

Reply:
Sorry, I must have been distracted. Change suggestion is updated. Keeping the `url` pointing to `Latest.zip` would produce a checksum mismatch as soon as you release a new version...

Reply:
That makes sense, thanks for the updated suggestion!